### ISSUE TITLE
Pressing and holding sliders  with keyboard makes particles dissappear

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/rutherford-scattering#126]
Found in #125.

**Comment by jessegreenberg** [comment]
Also, keyboard interaction with the tweaker buttons doesn't reset particles.

**Comment by jessegreenberg** [comment]
The problem with the sliders is the `removeFinger` and `addFinger` functions that are in place to fix multitouch issues.

**Comment by jessegreenberg** [comment]
Thanks for catching this @terracoda, should be fixed with the above commit.
